libs/api/domains/party-letter-registry/src/lib/partyLetterRegistry.service.ts

```diff
@@ -0,0 +1,43 @@
+import { Injectable } from '@nestjs/common'
+import { logger } from '@island.is/logging'
+import { ApolloError } from 'apollo-server-express'
+import {
+  PartyLetterRegistryApi,
+  PartyLetterRegistryControllerFindByManagerRequest,
+} from '../../gen/fetch/apis'
+import { CreateDto } from '../../gen/fetch'
+
+const handleError = async (error: any) => {
```

Comment:
Would be nice to have a shared util somewhere for that, I saw it used a couple of times

Reply:
Probably better to standardise this in a separate PR.
Will add this to my list